pkg/gadgets/trace/dns/types/dns.go

```diff
@@ -15,15 +15,25 @@
 package types
 
 import (
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	DNSName string `json:"name,omitempty"`
-	PktType string `json:"pktType,omitempty"`
-	QType   string `json:"qtype,omitempty"`
+	PktType string `json:"pktType,omitempty" column:"type,width:9,fixed"`
+	QType   string `json:"qtype,omitempty" column:"qtype,width:10,fixed"`
+	DNSName string `json:"name,omitempty" column:"name,width:24"`
+}
+
+func GetColumns() *columns.Columns[Event] {
+	cols := columns.MustCreateColumns[Event]()
+
+	col, _ := cols.GetColumn("container")
+	col.Visible = false
```

Comment:
As I suggested [here](https://github.com/kinvolk/inspektor-gadget/pull/980#discussion_r977633141). I think adding the API to perform these operations would be useful: `ColumnMap.SetVisible()`.

Reply:
I don't have a strong opinion. It seems too verbose to create all those setters, we can leave the member public.

Reply:
Yeah, let's keep the current implementation as also discussed [here](https://github.com/kinvolk/inspektor-gadget/pull/980#discussion_r981096998).